### ISSUE TITLE
Apple M1 Pro framework bug

### ISSUE THREAD

**Issue by NetMajom** [Redth/dotnet-maui-check#131]
Hello

I have installed everything well, manually also installed the required packages, but still have this issue

```
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '5.0.0' (x64) was not found.
  - The following frameworks were found:
      6.0.0-rc.2.21480.5 at [/usr/local/share/dotnet/x64/shared/Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The specified framework can be found at:
  - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=5.0.0&arch=x64&rid=osx.11.0-x64
   
 ```

Please help to solve this issue.

Or/if it cannot be solved, does it has any uninstalling sequence?

**Comment by NetMajom** [comment]
Do you know how/where can I set this?

> Uh the workload is for targeting .NET 6 so try something like `net6.0-maccatalyst` as a target framework and see if the issue goes away.

Do you know how/where can I set this?

**Comment by NetMajom** [comment]
> Open it in Notepad.

Yeah, its funny. What file? If you can't create a new project because the maui-check failed, with vs preview the whole project is missing, than open in notepad that file what you can't create, and what is not exist?

**I need the maui check command finish well.**

So, once again. The maui check command give me that error,  help me to solve, that the maui check command can be finish well.

**Comment by NetMajom** [comment]
> hmm I wonder if it's because the M1 is arm based and if so this tool might actually need to be retargeted to .NET 6 to fix it.

OK. Do you know any workaround when//how can it be retargeted?

**Comment by AraHaan** [comment]
Step 1: Fork this repository.
Step 2: Clone that fork.
Step 3: Open the projects in Visual Studio or Rider and change their target frameworks from .NET 5 to .NET 6 by right clicking each project and changing ``<TargetFramework>net5.0</TargetFramework>`` to ``<TargetFramework>net6.0</TargetFramework>``.
Step 4: Commit.
Step 5: Create Pull Request :smile:
